test/unit/generateUniqueId/generateUniqueId_test.js

```diff
@@ -0,0 +1,38 @@
+goog.module('measurementLibrary.testing.eventProcessor.generateUniqueId');
+goog.setTestOnly();
+
+const {generateUniqueId} = goog.require('measurementLibrary.eventProcessor.generateUniqueId');
+
+describe('The `generateUniqueId` method', () => {
+  const secondsSinceEpoch = 1234567890;
+  beforeAll(() => {
+    jasmine.clock().install();
+    jasmine.clock().mockDate(new Date(secondsSinceEpoch * 1000));
+  });
+
+  const uniqueIdFormat =
+      /^\d{1,10}.1234567890$/;
+
+  it('generates unique ID in correct format using Crypto API', () => {
+    for (let i = 0; i < 10; ++i) {
```

Comment:
I'm not confident that a loop adds any value here..  
Could you please instead spy on crypto and make verify it's called (and probably verify that Math.random has never been called)